tests/Functional/PHPCodesniffer/Standards/ZooRoyal/Sniffs/Commenting/DocCommentSniffTest.php

```diff
@@ -39,8 +39,8 @@ public function processApprovesCorrectCount()
     {
         $fileToTest = 'tests/Functional/PHPCodesniffer/Standards/ZooRoyal/'
             . 'Sniffs/Commenting/Fixtures/FixtureCorrectComments.php';
-        $subject = new Process($this->commandPrefix . $fileToTest, self::$vendorDir . '/../');
 
+        $subject = new Process(array_merge(explode(' ', $this->commandPrefix), [$fileToTest]), self::$vendorDir . '/../');
```

Comment:
Ist das vll eher was du meinst?

```suggestion
        $subject = new Process(array_push(explode(' ', $this->commandPrefix), $fileToTest), self::$vendorDir . '/../');
```

Reply:
^^so geht leider nicht, weil array_push nicht array returned, ich habe es aber umgeschrieben, dass es einheitlich mit [] gepushed wird.